src/cockpit/bridge.py

```diff
@@ -62,10 +63,15 @@ def __init__(self, args: argparse.Namespace):
         self.superuser_rule = SuperuserRoutingRule(self, args.privileged)
         self.internal_bus.export('/superuser', self.superuser_rule)
 
+        # we run this after the channel rules, so we don't need to check source="internal"
+        self.pcp_rule = PeerRoutingRule(self, ['/usr/libexec/cockpit-pcp'], {},
```

Comment:
The path needs to come from the manifest. But see below.

Reply:
See PR #18168 for actually fixing the manifest expansion of that macro.

Reply:
This is taken from the manifest now.